packages/perseus/src/mixins/changeable.ts

```diff
@@ -118,6 +113,26 @@ export const change: ChangeFn = function (
     );
 };
 
+export function changeSingleProp<
```

Comment:
Thinking about this further.  A change that would ripple through our codebase a bit more, but I think simplify this API is doing away with the diff between changing a single prop and multiple.  With object literals, I feel like there's almost no benefit of this: 

```js
Changeable.changeSingle(this, "offset", 3);
``` 

compared to

```js
Changeable.change(this, { offset: 3 });
```

In fact, I think the second is easier and clearer to read. 

So, could we do away with `changeSingle` and `changeMultiple` and just have `change(component, changedProps)`?

Reply:
I think that makes sense and I think we should be able to do this in a typesafe way now this function is parameteric.